docker-compose.example.mysql.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.7"
+services:
+  app:
+    image: babybuddy/babybuddy
+    # See README.md#configuration for other environment configuration options.
+    environment:
+      - ALLOWED_HOSTS=
+      - DJANGO_SETTINGS_MODULE=babybuddy.settings.mysql
+      - SECRET_KEY=
+      - TIME_ZONE=
+      - DEBUG=False # Turn to False in production
+    volumes:
+      - data:/app/data:rw
+      - media:/app/media:rw
+      - /path/to/my.cnf:/app/data/my.cnf
```

Comment:
Can this example just be `- data/my.cnf:/app/data/my.cnf` or does it need to be this way? I was kind of hoping we could guide people to just drop `my.cnf` _in to_ the `data` directory so it would be picked up there without an extra `volumes` line here.